streams/src/test/java/org/apache/kafka/streams/state/internals/SessionKeySchemaTest.java

```diff
@@ -144,18 +160,84 @@ public void testLowerBoundMatchesTrailingZeros() {
 
         assertThat(
             "appending zeros to key should still be in range",
-            lower.compareTo(
-                SessionKeySerde.bytesToBinary(
+            lower.compareTo(Bytes.wrap(
+                    SessionKeySchema.toBinary(
                     new Windowed<>(
                         Bytes.wrap(new byte[]{0xA, 0xB, 0xC, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}),
                         new SessionWindow(Long.MAX_VALUE, Long.MAX_VALUE))
                 )
-            ) < 0
+            )) < 0
         );
 
-        assertThat(lower, equalTo(SessionKeySerde.bytesToBinary(new Windowed<>(Bytes.wrap(new byte[]{0xA, 0xB, 0xC}), new SessionWindow(0, 0)))));
+        assertThat(lower, equalTo(Bytes.wrap(SessionKeySchema.toBinary(new Windowed<>(Bytes.wrap(new byte[]{0xA, 0xB, 0xC}), new SessionWindow(0, 0))))));
     }
 
+    @Test
+    public void shouldSerializeDeserialize() {
```

Comment:
Why do we test `SessionWindowedSerde` in this test class? Same for the next 3 tests.

Reply:
Seems it is a stale diff? `SessionWindowedSerde` has been completely removed.

Reply:
Well. The tested `keySerde` is a `SessionWindowSerde`

https://github.com/apache/kafka/pull/3307/files/0e3925a99857b1f17b9881d0b3aaaf5e4139d0db#diff-8650f75f1d517c7698d4d618d8cdf1a7R52

Thus, this test should go to `WindowedSerdesTest` IMHO.